src/main/java/com/spm/vasylyshyn/model/Device.java

```diff
@@ -44,22 +43,15 @@ public class Device {
     private List<DisplayCount> displayCounts = new ArrayList<>();
 
     @Column(updatable = false)
-    private String createdDate;
 
-    @PrePersist
-        protected void onCreate(){
-       createdDate =  ZonedDateTime
-                .now(
-                        ZoneId.of( "Europe/Bucharest" )
-                )
-                .format(
-                        DateTimeFormatter
-                                .ofLocalizedDateTime( FormatStyle.SHORT )
-                                .withLocale(
-                                        new Locale( "ro" , "RO" )   // Romanian in Romania.
-                                )
-                );
+//    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSXXX")
+//    @JsonDeserialize(using = LocalDateDeserializer.class)
+//    @JsonSerialize(using = LocalDateSerializer.class)
+    private LocalDateTime createdDate;
```

Comment:
```suggestion
    private LocalDateTime createdDate;
```
Don't submit commented out code chunks